nightly-examples/examples/spawny_thing.rs

```diff
@@ -0,0 +1,48 @@
+#![feature(async_await)]
+#![deny(rust_2018_idioms)]
+
+/// This is a example showing how information is scoped.
+///
+/// You can run this example by running the following command in a terminal
+///
+/// ```
+/// RUST_LOG=spawny_thing=trace cargo +nightly run --example spawny_thing
```

Comment:
Since we already set the verbosity level to `trace` in `main`, we don't need the `RUST_LOG` env var here. Could just be:
```suggestion
/// cargo +nightly run --example spawny_thing
```